### PR TITLE
Create enumerable

### PR DESCRIPTION
**Pull request by keroloussamy**
In this PR, we made the following 

- [ ] Create a class MyList that has an instance variable @list.
- [ ] In MyList implement a method #each that yields successive members of @list and uses the MyEnumerable module.
- [ ] Create a module MyEnumerable that implements the following methods: all?, any?, filter.

### CODE REVIEW

File: mylist.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require_relative 'myenumerable'
```
Comment (DammyShittu) [inline mylist.rb]: - Kindly change the file name that you're requiring above ☝️ to `my_enumerable` as that is the correct file name.
I am currently getting an error when I try to use the Interactive Ruby Shell(IRB).

Here 👇 is a screenshot of what I got when I use the IRB

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Lucas-Masaba @keroloussamy, 👋 

Superb work done so far. 👏

Kudos to you. 👍

There is an issue that you still need to work on to go to the next milestone but, trust me, you are ALMOST THERE!

# Required Changes ♻️

_Please check the comments under the review._

# Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mylist.rb (hunk 1 above)
Comment (Lucas-Masaba) [reply mylist.rb]: Thank you for the tip. I have fixed the issue

**Review by oloomoses — APPROVED** [review]
Hi team, 

Well done on making the suggested changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**